src/sfctl/custom_cluster.py

```diff
@@ -30,10 +31,15 @@ def select_arg_verify(endpoint, cert, key, pem, ca, no_verify):
     if any([cert, key]) and not all([cert, key]):
         raise CLIError(usage)
 
+    if aad and any([pem, cert, key]):
+        raise CLIError(usage)
+
     if pem and any([cert, key]):
         raise CLIError(usage)
 
-def select(endpoint, cert=None, key=None, pem=None, ca=None, no_verify=False):
+def select(endpoint, cert=None, key=None, pem=None, ca=None,
+           aad=False, no_verify=False):
+    #pylint: disable-msg=R0914
```

Comment:
Convention is to use the string so `pylint disable=too-many-local-vars` I believe.